.github/workflows/ci.yml

```diff
@@ -31,6 +57,5 @@ jobs:
           toolchain: nightly
           components: rust-src
 
-      - run: cargo build --release
-        working-directory: ${{ matrix.project }}
-
+      - run: cargo build --release --example ${{ matrix.project.name }}
```

Comment:
I guess we could also just use `cargo build --release --examples` but this certainly also works

Reply:
Good idea! Just updated it to use `examples`